src/components/App/App.tsx

```diff
@@ -111,27 +116,51 @@ class App extends React.Component<Props> {
     this.setState({ savedNotes: [...this.state.savedNotes, {name: ApiName, notes: [note]}]});
   }
 
+  deleteNote = (apiName: string, content: string) => {
+    const myNote = this.state.savedNotes.find(note => note.name === apiName)
+    const unEditedNotes = this.state.savedNotes.filter(note => note !== myNote)
+    myNote?.notes.splice(myNote.notes.indexOf(content), 1)
+    this.setState({ savedNotes: [...unEditedNotes, myNote]})
+  }
+
   render() {
     this.state.favorites.length && localStorage.setItem('favorites', JSON.stringify(this.state.favorites));
     this.state.savedNotes.length && localStorage.setItem('notes', JSON.stringify(this.state.savedNotes));
     return (
       <div className="App">
         <SideBar></SideBar>
         <Route exact path='/' render={() => {
-          return <main><h1>metAPI</h1><FilterForm filter={this.filter} apiList={this.state.apiList}/>
-          <CardContainer apiList={this.state.currentApis}></CardContainer></main>
+          return (
+            <main>
+              <h1>metAPI</h1>
+              <FilterForm filter={this.filter} apiList={this.state.apiList}/>
+              <CardContainer 
+                apiList={this.state.currentApis} 
+                toggleFavorite={this.toggleFavorite} 
+                favorites={this.state.favorites}
+              />
+            </main>
+          )
         }}/>
         <Route
           path="/:title"
           render={({ match }) => {
```

Comment:
might be worth breaking this out into it's own function to clean up the return a little bit